tests/phpunit/integration/SkinCitizenPreferenceDefaultsTest.php

```
@@ -0,0 +1,138 @@
+
+/**
+ * Client preference default classes stamped onto <html>, including
+ * defaults read from MediaWiki:Citizen-preferences.json through
+ * MessageCache. Editing the page with real JsonContent also proves the
+ * MessageCache path serves JSON content-model pages.
+ *
+ * @group Citizen
+ * @group Database
+ * @covers \MediaWiki\Skins\Citizen\SkinCitizen
+ */
+class SkinCitizenPreferenceDefaultsTest extends MediaWikiIntegrationTestCase {
```

PHPStan (2.2.7)

[error] Class MediaWiki\Skins\Citizen\Tests\Integration\SkinCitizenPreferenceDefaultsTest extends unknown class MediaWikiIntegrationTestCase.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)